<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComponentPreview</class>
 <widget class="QWidget" name="ComponentPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>389</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <item row="18" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="labelTitle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Resistor</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="editText">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QRadioButton" name="btnIndependent">
         <property name="text">
          <string>Independent</string>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <layout class="QGridLayout" name="anchorPicker">
         <property name="spacing">
          <number>2</number>
         </property>
        </layout>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="labelTextX">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>X:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLineEdit" name="editTextY">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QRadioButton" name="btnDepend">
         <property name="text">
          <string>Depend</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="labelTextAngle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Angle:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="labelTextPos">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Position:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="editAngle">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLineEdit" name="editTextX">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="Line" name="sepText">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="labelText">
         <property name="text">
          <string>Text:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="labelAnchor">
         <property name="text">
          <string>Anchor:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelAngle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Angle:</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QLineEdit" name="editTextAngle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="Line" name="sepTitle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="labelTextY">
         <property name="text">
          <string>Y:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelValue">
         <property name="text">
          <string>Value:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editValue">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="preview"/>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="btnAdd">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
